pkg/controller/stats/clickhouse_stats.go

```diff
@@ -0,0 +1,228 @@
+// Copyright 2022 Antrea Authors.
```

Comment:
Just for curiosity, why this part is put in controller instead of apiserver pkg?

Reply:
Thanks for the reviewing. It shouldn't be there as it doesn't contain functionality of controller. I've moved it to apiserver/utils/stats.